docs/api/public_php_api_search.md

```diff
@@ -87,6 +87,35 @@ As such, `query` is recommended when the search is based on user input.
 The difference between `query` and `filter` is only relevant when using Solr search engine.
 With the Legacy search engine both properties will give identical results.
 
+#### Batch searching
+
+To search in a large result set, use `eZ\Publish\API\Repository\Iterator\BatchIterator`.
+`BatchIterator` divides the result set into smaller batches, which helps with sets that are too large to handle due to memory limits.
```

Comment:
`eZ\Publish\API\Repository\Iterator\BatchIterator` allows to iterate over all search or filtering results within memory constrains by automatically dividing the results into smaller batches (pages).

Reply:
```suggestion
To search in a large result set, use `eZ\Publish\API\Repository\Iterator\BatchIterator`.
`BatchIterator` divides the results of search or filtering into smaller batches.
This enables iterating over results that are too large to handle due to memory constraints.
```